content/en/network_performance_monitoring/installation.md

```diff
@@ -122,14 +126,52 @@ If you need to use Network Performance Monitoring on other systems with SELinux
 
 If these utilities do not exist in your distribution, follow the same procedure but using the utilities provided by your distribution instead.
 
+### Windows systems
+
+Data collection for Windows systems is done using a Microsoft-certified Windows driver, which is now available in public beta for versions 2016 or later. Note that the driver currently monitors Windows hosts only, and not Windows containers. DNS metric collection is not currently supported for Windows systems.
```

Comment:
Same.   Can't say "Microsoft certified".
Suggest:

Data collection for windows systems is now available for Windows versions 2016 and later.  Note that monitoring Windows hosts only is currently supported; not Windows containers.  DNS metric collection is not currently supported for Windows systems.

Reply:
fixed!